src/components/config/index.js

```diff
@@ -9,6 +9,7 @@ export const TOPICS = {
   android: 'Android',
   data: 'Data / AI',
   php: 'PHP',
+  'dev-ops': 'Dev Ops',
```

Comment:
`devops` instead of `dev-ops` ?

Reply:
I wasn't sure. But I've checked what StackOverflow used and it's `#devops` so I'll update that 👍 Thanks!